src/Util/XML.php

```diff
@@ -141,9 +141,9 @@ public static function fromArray(array $array, &$xml, bool $remove_header = fals
 	 * @param string        $elementname Name of the XML element of the target
 	 * @return void
 	 */
-	public static function copy(&$source, &$target, string $elementname)
```

Comment:
What kind of types are possible for elementname? What's the source for it? Why not mapping/checking it before calling a copy function?

For me, moving such checks near to the source makes the reason more transparent for others.. and maybe leads to refactorings :D

Reply:
This PR just restores the original function before the type hints had been added. Currently it is effectively blocking the processing of incoming messages.

I'm sure that we can rewrite that whole part in a better way. But for now this PR fixes a critical issue.